### DISCUSSION TITLE
Github actions workflow went into queued state even after 20 hours of workflow push event creation.

### DISCUSSION THREAD

**Discussion by charan-happy** [community/community#112514] [Actions] [answered]
### Select Topic Area

Question

### Body

Hello team,

I am new to GitHub actions, I created a new workflow and it went into a queued state (I don't know why). it was in the same state even after 20 hours and also the same status. How should I get it passed through?

Please have a look at below attached image to understand my issue further.

![image](https://github.com/community/community/assets/89054489/9e1fd715-bffa-44ce-983e-8ab8449b6bc8)

**Comment by Bulat-Ziganshin** [comment, marked as answer]
i had this problem when I incorrectly wrote "runs-on" property, so it waited for non-existent host type

**Reply by charan-happy** [reply]
Yes you are correct @Bulat-Ziganshin . I too did the same mistake .

Under runs-on instead of ubuntu-latest I gave Ubuntu:latest.
